loader/plugins/prune.js

```diff
@@ -0,0 +1,42 @@
+const CordlrPlugin = require('cordlr-plugin')
+
+module.exports = class HelpPlugin extends CordlrPlugin {
+  constructor (bot, config) {
+    super(bot, config)
+
+    this.name = 'Cordlr Core - Prune'
+    this.description = 'Cordlr Core plugin for pruning an specific amount of messages'
+
+    this.commands = {
+      'prune': {
+        'usage': '<x>',
+        'function': 'pruneMessages',
+        'description': 'Prunes the last x messages',
+        'permissions': [
+          'MANAGE_MESSAGES'

```

Comment:
Yes with the Administrator permission everything run smoothly, but I am wondering if the plugin should require the "Administrator" permission instead of "Manage_Messages".

Without the Administrator permission it can NOT delete what an Administrator writes.

Example _without_ Administrator Permission:
Line 1: **User**: "some comment"
Line 2: **Administrator**: "some comment"
Line 3: **User**: "some comment"
Line 4: **Administrator**: *!prune N*

!prune 1 will try and delete line 4 but **fail.**
!prune 2 will try and delete line 4 **(fail)** and line 3 **(succeed)**.
!prune 3 will try and delete line 4 **(fail)**, line 3 **(succeed)** and line 2 **(fail)**.
!prune 4 will try and delete line 4 **(fail)**, line 3 **(succeed)**, line 2 **(fail)** and line 1 **(succeed)**.